README.md

```diff
@@ -179,19 +179,38 @@ earlier builds do not affect the new build.
 Fortran Package Manager (fpm) is a package manager and build system for Fortran.   
 You can build `stdlib` using provided `fpm.toml`:
 
+**Option 1**: From root folder
+
+As `fpm` does not currently support `fypp` natively, `stdlib` now proposes a python script to preprocess and build. This script enables modification of the different `fypp` macros available in `stdlib`. The preprocessed files will be dumped at `<current_folder>/temp/*.f90` or `*.F90`. You can use the following command line:
+
 ```sh
-git checkout stdlib-fpm
+python config/fypp_deployment.py
 fpm build --profile release
 ```
 
-**Alternative**: as `fpm` does not currently support `fypp` natively, building `stdlib` with `fpm` can be done in two steps: a) launch the preprocessor through the `fpm-deployment.sh` script, which creates a subfolder `stdlib-fpm` and b) build the project using the processed files within the latter subfolder. This process can be done with the following commands:
+or the short-cut
 
 ```sh
-source ./ci/fpm-deployment.sh
-cd stdlib-fpm/
-fpm build --profile release
+python config/fypp_deployment.py --build 1
```

Comment:
Why is the `1` needed?
could it be:
```suggestion
python config/fypp_deployment.py --build
```

Reply:
Since I defined the parameter as a boolean, it expects a value (an int that can be 0 or 1) after the key word, otherwise it throws an error, I can check if there is a way to facilitate that.

Reply:
found it https://stackoverflow.com/questions/8259001/python-argparse-command-line-flags-without-arguments and solved with this commit https://github.com/fortran-lang/stdlib/pull/802/commits/95ee064fa3d61d5b6d4107bface789af6d6ae462